lib/minasu/db.py

```diff
@@ -1,22 +1,23 @@
-import os,shutil
 from minasu import settings
+import os,shutil
+
 
 class db():
     def __init__(
         self,
-        instance_name, 
+        instance_name,
         instance_dir=settings.default_settings["instance_dir"]
     ):
         self.instance_name = instance_name
         self.instance_dir  = instance_dir
         self.instance_path = self.instance_dir + "/" + self.instance_name
 
-    ## create a new db at folder location
+    # create a new db at folder location
     def create(self):
         data = {}
-        #data["instance_name"] = self.instance_name
-        #data["instance_dir"]  = self.instance_dir + "/"
-        #data["instance_path"] = self.instance_path + "/"
+        # data["instance_name"] = self.instance_name
```

Comment:
this is another dumb linter mistake.  These aren't comments (which should have a space after the #), but commented code.  :man_shrugging: